pkgs/development/libraries/science/math/suitesparse/no_librt_mac.patch

```diff
@@ -0,0 +1,15 @@
+diff -ur SuiteSparse/SuiteSparse_config/SuiteSparse_config.mk MySuite/SuiteSparse_config/SuiteSparse_config.mk
```

Comment:
This patch is simple enough that it could be replaced by a `sed` call or `substituteInPlace`. Would save a few kbs. What do you think?